packages/theme/src/themes/troms-theme/theme.ts

```diff
@@ -191,13 +191,13 @@ const themes: Themes = {
       },
     },
     transport: {
-      transport_city: {
-        primary: baseColors.green_700,
-        secondary: baseColors.green_800,
-      },
       transport_region: {
+        primary: baseColors.green_800,
+        secondary: baseColors.green_900,
+      },
+      transport_city: {
         primary: baseColors.blue_400,
-        secondary: baseColors.blue_400,
+        secondary: baseColors.blue_500,
```

Comment:
You should run `yarn build-all` to apply these changes to the .css-files and include the changes to this PR 👍